manifests/jenkins.yaml

```diff
@@ -127,6 +131,11 @@ objects:
           secret:
             secretName: slack-api-token
             optional: true
+        # DELTA: add AWS creds
+        - name: aws-fcos-builds-bot-creds
+          secret:
+            secretName: aws-fcos-builds-bot-config
+            optional: true
```

Comment:
should we add a comment here about why we're adding them (i.e. specifically so we can have them for the sync-stream-metadata.Jenkinsfile). Otherwise one might get confused about the source of truth for the configs for the actual build uploads (in pod.yaml)